### DISCUSSION TITLE
Is variant type 'number' supported?

### DISCUSSION THREAD

**Discussion by pc-pallon** [Unleash/unleash#6954] [Q&A] [answered]
In the code such as [here](https://github.com/Unleash/unleash/blob/49e84d3a911a1fc66c374cd97d63a6554092222e/src/lib/types/model.ts#L140) and in the UI such as in the screenshot below, the variant type 'number' seems to be supported.
<img width="801" alt="Screenshot 2024-04-26 at 14 25 01" src="https://github.com/Unleash/unleash/assets/143699616/1581725c-e632-4265-8640-b73c8171132f">

However, in other code locations such as [here](https://github.com/Unleash/unleash/blob/49e84d3a911a1fc66c374cd97d63a6554092222e/frontend/src/utils/variants.ts#L1) and in the document [here](https://docs.getunleash.io/reference/feature-toggle-variants#variant-payload), there is no type 'number'.

Is it actually supported?

**Comment by FredrikOseberg** [comment, marked as answer]
Hi @pc-pallon. Apologies for the late response. The short answer is that the type property is merely a hint to your own application on how you should parse the content. Unleash does not perform the parsing for you. I have added a note in the documentation about this here: https://docs.getunleash.io/reference/feature-toggle-variants#variant-payload

Thanks for calling out the confusion!

**Reply by pc-pallon** [reply]
Got it! Thanks for the explanation and the updated documentation 😄
